### PR TITLE
GG-25964 Fixed authorization of an user with not notmalized email.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/e2e/testcafe/page-models/pageAdminListOfRegisteredUsers.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import {Selector} from 'testcafe';
 import {Table} from '../components/Table';
 
 export const TEST_USER = {
-    email: 'foo@example.com',
+    email: 'N3wUs3r@D0main.c0m',
```
[the remark this answers is not in the document]
Comment (vsisko) [reply modules/web-console/e2e/testcafe/page-models/pageAdminListOfRegisteredUsers.js]: It is the special user for "Become this user" test.

File: modules/web-console/e2e/testcafe/environment/envtools.js [hunk 2]
```diff
@@ -39,15 +39,14 @@ const insertTestUser = (user = testUser) => {
         .catch((err) => {throw err.message;});
 };
 
-const dropTestDB = (user) => {
+const dropTestDB = (user = testUser) => {
```
[the remark this answers is not in the document]
Comment (vsisko) [reply modules/web-console/e2e/testcafe/environment/envtools.js]: Applied fixes for drop DB function with argument.

Comment (nva) [inline modules/web-console/e2e/testcafe/environment/envtools.js]: Please use same signature as in https://github.com/gridgain/gridgain/pull/727/files#r351979491

Comment (vsisko) [reply modules/web-console/e2e/testcafe/environment/envtools.js]: Changed argument to email mask.

**Pull request merged**